automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/rstudio/RShinyPage.scala

```diff
@@ -0,0 +1,22 @@
+package org.broadinstitute.dsde.workbench.leonardo.rstudio
+
+import org.broadinstitute.dsde.workbench.auth.AuthToken
+import org.broadinstitute.dsde.workbench.page.CookieAuthedPage
+import org.openqa.selenium.WebDriver
+
+import scala.concurrent.duration._
+
+class RShinyPage(val url: String)(implicit override val authToken: AuthToken, implicit val webDriver: WebDriver)
+    extends CookieAuthedPage[RShinyPage] {
+  private val exampleHeader: Query = cssSelector("#showcase-app-container > div > h2")
```

Comment:
css selectors tend to be pretty brittle... especially since this seems to rely on css in a third party app.

Sometimes it's the best way, but definitely worth thinking about for a second.

Reply:
Yeah - think this works for the "example" apps from the shiny package, but definitely wouldn't work for all shiny apps. Also would break if shiny changes their examples. Can't really think of a better way either though..